src/grpc/tests/proto/news_enum.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package news;
+
+enum Status {
+    PUBLISHED = 0;
+    DRAFT = 1;
+}
+
+message News {
+    int32 id = 1;
+    string title = 2;
+    string body = 3;
+    string postImage = 4;
+    Status foo = 5;
+    message Author { // nested field
+        string name = 1;
+        string email = 2;
+    }
+    Author author = 6;
+}
+
+// my awesome doc for service
+service NewsService {
+    rpc GetAllNews (google.protobuf.Empty) returns (NewsList) {}
+    rpc GetNews (NewsId) returns (News) {}
+    rpc GetMultipleNews (MultipleNewsId) returns (NewsList) {}
+    rpc DeleteNews (NewsId) returns (google.protobuf.Empty) {}
+    rpc EditNews (News) returns (News) {}
+    rpc AddNews (News) returns (News) {}
```

Comment:
Need tests for 
- [ ] A test for a type used as both input and output
- [ ] A test for mutation
- [ ] A test for empty objects
- [ ] A test for two types having same name in different packages